### ISSUE TITLE
The `#default-search` input should have a label element

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-spine#402]
This can be a screen reader only element via class `.screen-reader-text`, but it needs to exist.

See: https://www.w3.org/TR/WCAG20-TECHS/F68
